api/catalog/api/utils/pagination.py

```diff
@@ -42,6 +45,8 @@ def page(self, value):
         value = int(value)  # convert str params to int
         if value <= 0:
             raise get_api_exception("Page must be greater than 0.", 400)
+        elif value > 20:
+            raise get_api_exception("Searches are limited to 20 pages.", 400)
```

Comment:
```suggestion
            raise get_api_exception("Searches are limited to the first 20 pages of results while we mitigate API performance issues. We are working on a solution for bulk exports of Openverse's data.", 400)
```

Might want to be super verbose here? Not sure what else we could say 🤷 CC @WordPress/openverse-maintainers if anyone else has ideas. Would we, for example, want to tell them to email us or anything? Not sure.

Reply:
Maybe we could just say `"Searches are currently limited to 20 pages"`, which implies that might change in the future?